elasticsearch.yml

```diff
@@ -4,6 +4,7 @@ auxiliary_index_names: ["page-traffic", "metasearch"]
 registry_index: "government"
 metasearch_index_name: "metasearch"
 popularity_rank_offset: 10
+snapshot_bucket_name: "govuk-api-elasticsearch-snapshots-integration"
```

Comment:
Yes. Yes it should. We will have to make a corresponding change to the puppet for this, but we can do that after.